### ISSUE TITLE
Python packaging of tool including database

### ISSUE THREAD

**Issue by peterjc** [peterjc/thapbi-pict#115]
Meta issue for getting a working package up on PyPI.

**Comment by peterjc** [comment]
Testing locally, making a tar-ball is working nicely with both the HMM files (placed within the Python code) and ``database/ITS1_DB.sql`` present - not due to having recently edited the manifest this was needed first:

```
$ rm -rf  thapbi_pict.egg-info
```

Then,

```
$ python setup.py sdist --formats=gztar
...
$ cd dist
$ $ tar -zxvf thapbi_pict-0.1.5.tar.gz 
thapbi_pict-0.1.5/
thapbi_pict-0.1.5/CONTRIBUTING.md
thapbi_pict-0.1.5/LICENSE.md
thapbi_pict-0.1.5/MANIFEST.in
thapbi_pict-0.1.5/PKG-INFO
thapbi_pict-0.1.5/README.md
thapbi_pict-0.1.5/database/
thapbi_pict-0.1.5/database/ITS1_DB.sql
thapbi_pict-0.1.5/database/README.md
thapbi_pict-0.1.5/setup.cfg
thapbi_pict-0.1.5/setup.py
thapbi_pict-0.1.5/tests/
thapbi_pict-0.1.5/tests/assess/
thapbi_pict-0.1.5/tests/assess/assess_sample.tsv
...
thapbi_pict-0.1.5/tests/test_seq-import.sh
thapbi_pict-0.1.5/thapbi_pict/
thapbi_pict-0.1.5/thapbi_pict/__init__.py
thapbi_pict-0.1.5/thapbi_pict/__main__.py
thapbi_pict-0.1.5/thapbi_pict/assess.py
thapbi_pict-0.1.5/thapbi_pict/classify.py
thapbi_pict-0.1.5/thapbi_pict/db_import.py
thapbi_pict-0.1.5/thapbi_pict/db_orm.py
thapbi_pict-0.1.5/thapbi_pict/dump.py
thapbi_pict-0.1.5/thapbi_pict/hmm/
thapbi_pict-0.1.5/thapbi_pict/hmm/__init__.py
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.fas
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.hmm
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.hmm.h3f
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.hmm.h3i
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.hmm.h3m
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.hmm.h3p
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.sto
thapbi_pict-0.1.5/thapbi_pict/hmm/phytophthora_its1.txt
thapbi_pict-0.1.5/thapbi_pict/legacy.py
thapbi_pict-0.1.5/thapbi_pict/ncbi.py
thapbi_pict-0.1.5/thapbi_pict/prepare.py
thapbi_pict-0.1.5/thapbi_pict/sample_summary.py
thapbi_pict-0.1.5/thapbi_pict/seq_import.py
thapbi_pict-0.1.5/thapbi_pict/summary.py
thapbi_pict-0.1.5/thapbi_pict/taxdump.py
thapbi_pict-0.1.5/thapbi_pict/utils.py
thapbi_pict-0.1.5/thapbi_pict.egg-info/
thapbi_pict-0.1.5/thapbi_pict.egg-info/PKG-INFO
thapbi_pict-0.1.5/thapbi_pict.egg-info/SOURCES.txt
thapbi_pict-0.1.5/thapbi_pict.egg-info/dependency_links.txt
thapbi_pict-0.1.5/thapbi_pict.egg-info/entry_points.txt
thapbi_pict-0.1.5/thapbi_pict.egg-info/requires.txt
thapbi_pict-0.1.5/thapbi_pict.egg-info/top_level.txt
```

This has the test files, and plain text ``database/ITS1_DB.sqlite`` ready to be able to run``sqlite3 ITS1_DB.sqlite < ITS1_DB.sql`` as part of a local install.

However, currently the wheel does not include ``database/ITS1_DB.sql``

```
$ python setup.py bdist_wheel
...
$ cd dist
$ unzip  thapbi_pict-0.1.5-py3-none-any.whl  
Archive:  thapbi_pict-0.1.5-py3-none-any.whl
  inflating: thapbi_pict/__init__.py  
  inflating: thapbi_pict/__main__.py  
  inflating: thapbi_pict/assess.py   
  inflating: thapbi_pict/classify.py  
  inflating: thapbi_pict/db_import.py  
  inflating: thapbi_pict/db_orm.py   
  inflating: thapbi_pict/dump.py     
  inflating: thapbi_pict/legacy.py   
  inflating: thapbi_pict/ncbi.py     
  inflating: thapbi_pict/prepare.py  
  inflating: thapbi_pict/sample_summary.py  
  inflating: thapbi_pict/seq_import.py  
  inflating: thapbi_pict/summary.py  
  inflating: thapbi_pict/taxdump.py  
  inflating: thapbi_pict/utils.py    
  inflating: thapbi_pict/hmm/__init__.py  
  inflating: thapbi_pict/hmm/phytophthora_its1.fas  
  inflating: thapbi_pict/hmm/phytophthora_its1.hmm  
  inflating: thapbi_pict/hmm/phytophthora_its1.hmm.h3f  
  inflating: thapbi_pict/hmm/phytophthora_its1.hmm.h3i  
  inflating: thapbi_pict/hmm/phytophthora_its1.hmm.h3m  
  inflating: thapbi_pict/hmm/phytophthora_its1.hmm.h3p  
  inflating: thapbi_pict/hmm/phytophthora_its1.sto  
  inflating: thapbi_pict/hmm/phytophthora_its1.txt  
  inflating: thapbi_pict-0.1.5.dist-info/LICENSE.md  
  inflating: thapbi_pict-0.1.5.dist-info/METADATA  
  inflating: thapbi_pict-0.1.5.dist-info/WHEEL  
  inflating: thapbi_pict-0.1.5.dist-info/entry_points.txt  
  inflating: thapbi_pict-0.1.5.dist-info/top_level.txt  
  inflating: thapbi_pict-0.1.5.dist-info/RECORD 
```

With the wheel file, ``database/ITS1_DB.sql`` is not included (along with lots of other files including the tests). What we really want here is a bundled ``ITS1_DB.sqlite`` (binary file) in the wheel.

Perhaps the simplest option is to run ``sqlite3 ITS1_DB.sqlite < ITS1_DB.sql`` as part of the release process, and put this file under the Python source tree (as done with the HMM files already)?

**Comment by peterjc** [comment]
Keeping this open until up on PyPI...

**Comment by peterjc** [comment]
Closed with the release of v0.1.6 on PyPI
